packages/core/src/routes/session.ts

```diff
@@ -318,7 +318,12 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
       }),
     }),
     async (ctx, next) => {
+      const { jti } = await provider.interactionDetails(ctx.req, ctx.res);
+      ctx.log.sessionId = jti;
+
       const { username, password } = ctx.guard.body;
+      ctx.log.type = LogType.RegisterUsernamePassword;
+      ctx.log.username = username;
```

Comment:
Can we infer the rest of the log properties by giving a specific LogType? Looks like there is no property guard or type hint so far.

Reply:
Added the linear issue LOG-2200